sdks/python/apache_beam/transforms/ptransform_test.py

```diff
@@ -2062,6 +2062,28 @@ def test_type_inference_command_line_flag_toggle(self):
     x = self.p | 'C2' >> beam.Create([1, 2, 3, 4])
     self.assertEqual(int, x.element_type)
 
+  def test_eager_execution(self):
+
+    class DoubleDoFn(beam.DoFn):
```

Comment:
Nit (here and below): Unless an actual DoFn is needed, I avoid the boilerplate and just use Map/FlatMap (possibly even with an inline lambda).